CMakeLists.txt

```diff
@@ -92,39 +92,46 @@ include_directories(${VOTCA_CSG_INCLUDE_DIRS})
 ########################################################################
 # Checks what linear algebra packages are installed                    #
 ########################################################################
-option(WITH_LINALG "Checking which BLAS libraries are installed" ON)
-if (WITH_LINALG)
 
-  find_package(Eigen)
 
 
-  if (EIGEN_FOUND) 
-    message("-- EIGEN found (not used at the moment)")
-    set(EIGEN TRUE) #used in votca_xtp_config.h
-    include_directories(${EIGEN_INCLUDE_DIRS})
-  else()
-    message("-- EIGEN not found")
-    set(EIGEN FALSE) #used in votca_xtp_config.h
-  endif()
+set(MKL FALSE)
+set(GSL FALSE)
 
-  set(MKL FALSE)
-  set(GSL FALSE)
+find_package(GSL)
+find_package(MKL)
 
-  #find_package(GSL)
-  find_package(MKL)
 
+if (MKL_FOUND AND CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
+message("-- MKL found, overloading uBLAS prod()")
+set(MKL TRUE) #used in votca_config.h
+elseif(GSL_FOUND)
+message("-- GSL found, overloading uBLAS prod()")
+include_directories(${GSL_INCLUDE_DIRS})
+set(GSL_PKG "gsl")
+set(GSL TRUE) #used in votca_config.h
+endif()
+
+if(MKL_FOUND)
+  set(MKL_URL "http://software.intel.com/sites/default/files/managed/76/8c/intel-mkl-and-boost-example.zip")
+  set(MKL_MD5 B957EE63A2167AD21B650B47726CD55C )
+  set(MKL_FILE "intel-mkl-and-boost-example.zip")
+  
+  include(ExternalProject)
+  ExternalProject_add(intel
```

Comment:
okay,  bjoern agrees. We will try use tools as much as possible. So I ll remove that part and simply import the tools prod header